main.py

```diff
@@ -1,6 +1,6 @@
 import discord
 import os
-from commands import createReminder, listUserReminders
+from commands import COMMAND_LIST
```

Comment:
Where is `COMMAND_LIST` from? Can't seem to find it in `commands.py`.

Reply:
good catch, fixed